old-ui/app/components/app-bar/main-menu.js

```diff
@@ -89,8 +89,8 @@ const mapDispatchToProps = dispatch => {
     showConfigPage: () => dispatch(actions.showConfigPage()),
     lockMetamask: () => dispatch(actions.lockMetamask()),
     showInfoPage: () => dispatch(actions.showInfoPage()),
-    showDomainsPage: () => dispatch(rifActions.showDomainsPage()),
-    showPaymentsPage: () => dispatch(rifActions.showPaymentsPage()),
+    showDomainsPage: () => dispatch(rifActions.navigateTo('domains')),
```

Comment:
do you think it makes sense to move those screen names to constants? if we are going to navigate to "domains" from another views it would be nice.

Reply:
Yes that's right